doc/compliancetest.rst

```diff
@@ -183,21 +187,27 @@ Connection of the switch ports is described in the table below:
 The Automated RT tester will detect "Device B" by itself. No configuration is
 required in the Automated RT tester menu.
 
+The switch can send its portID as ``port001.b`` instead of ``port001``, if the
```

Comment:
Maybe the port names used are "port-001"  ?

Reply:
Done